ui/org.eclipse.pde.ui/src/org/eclipse/pde/internal/ui/views/imagebrowser/filter/SizeFilter.java

```diff
@@ -36,31 +36,21 @@ public SizeFilter(final int width, final int widthType, final int height, final
 
 	@Override
 	public boolean accept(final ImageElement element) {
-		boolean accept = true;
-
-		switch (mWidthType) {
-			case TYPE_EXACT :
-				accept &= (element.getImageData().width == mWidth);
-				break;
-			case TYPE_BIGGER_EQUALS :
-				accept &= (element.getImageData().width >= mWidth);
-				break;
-			case TYPE_SMALLER_EQUALS :
-				accept &= (element.getImageData().width <= mWidth);
-				break;
-		}
-
-		switch (mHeightType) {
-			case TYPE_EXACT :
-				accept &= (element.getImageData().height == mHeight);
-				break;
-			case TYPE_BIGGER_EQUALS :
-				accept &= (element.getImageData().height >= mHeight);
-				break;
-			case TYPE_SMALLER_EQUALS :
-				accept &= (element.getImageData().height <= mHeight);
-				break;
-		}
+		
+
+		boolean accept = true & switch (mWidthType) {
```

Comment:
The leading `true` is not necessary here.

And the parenthesis in the following cases can probably be deleted too.